base/acme/src/org/dogtagpki/acme/ACMEAuthorization.java

```diff
@@ -35,6 +45,23 @@ public void setID(String id) {
         this.id = id;
     }
 
+    public String getAccountID() {
+        return accountID;
+    }
+
+    public void setAccountID(String accountID) {
+        this.accountID = accountID;
+    }
+
+    public Date getExpirationTime() {
+        return expirationTime;
```

Comment:
Also, is expiration time checked anywhere yet?

Reply:
Hmm.. good catch! Let me update the patch.

Reply:
I think we could also extend this to a "validator" interface and if the validated is expirable, have it check expiration too. 

But perhaps the term is perishable. XD